service/src/main/java/bio/terra/landingzone/stairway/flight/create/AwaitCreateLandingResourcesZoneFlightStep.java

```diff
@@ -14,7 +14,7 @@
 import java.util.Optional;
 
 public class AwaitCreateLandingResourcesZoneFlightStep implements Step {
-  public static final int FLIGHT_POLL_SECONDS = 1;
+  public static final int FLIGHT_POLL_SECONDS = 5;
```

Comment:
Since this value has been increased, doesn't `FLIGHT_POLL_CYCLES` need to decrease?

Reply:
I think we still need to keep the same amount of poll cycles. So, basically we are limiting time for sub-flight with 30 min. In includes time for rollback in case we have issues.

Reply:
You are right! Please disregard my initial response.